packages/model/src/entities/BlockNode/index.ts

```diff
@@ -78,6 +78,21 @@ export class BlockNode {
     this.#initialize(data);
   }
 
+  /**
+   * Allows accessing Block name
+   */
+  public get name(): string {
+    return this.#name;
+  }
+
+  /**
+   * Allows accessing Block data
+   */
+  public get data(): BlockNodeData {
```

Comment:
We should make the return type readonly (would be nice to check if it works for nested structures as well)